inference-engine/src/gna_plugin/runtime/pwl.h

```diff
@@ -23,7 +23,7 @@
 #define PWL_MAX_ERR_PERCENT 1.0f
 #define PWL_MAX_ITERATIONS_DEFAULT 2000
 #define PWL_MAX_ITERATIONS_LOG 5000
-#define PWL_MAX_NUM_SEGMENTS 128
+#define PWL_MAX_NUM_SEGMENTS (128-2)
```

Comment:
what is the magic number 2? Looks like a debug change

Reply:
In some cases, two segments are added to the beginning and end when creating a PWL on the GNU side (see create_multi segment_gna_pwl() in inference-engine/src/gna_plugin/backend/make_pwl.cpp ).